lib/bike_brigade_web/live/campaign_signup_live/show.ex

```diff
@@ -110,6 +110,30 @@ defmodule BikeBrigadeWeb.CampaignSignupLive.Show do
     {:noreply, socket}
   end
 
+  def handle_event("signup_rider", %{"rider_id" => rider_id, "task_id" => task_id}, socket) do
+    %{campaign: campaign, tasks: tasks} = socket.assigns
+    task = Enum.find(tasks, fn task -> task.id == task_id end)
+
+    attrs = %{
+      # this is arbitrary and not actually used by dispatchers anymore.
+      "rider_capacity" => "1",
+      # this will need to be configurable at some point, for campaigns that have pickup time edge cases.
+      "pickup_window" => "no time specififed",
```

Comment:
We can make this the pickup window that covers the whole campaign, take a look at e.g. https://github.com/bikebrigade/dispatch/blob/24931a0541fbc08f4ff15f0bb7cf267a4ef48509/lib/bike_brigade_web/live/campaign_signup_live/show.html.heex#L14 for a place we generate that string